main.go

```diff
@@ -173,7 +191,37 @@ var (
 		newPerThreadCounterMetric("flow_wrk", "flows_evicted_pkt_inject_total", "", "flows_evicted_pkt_inject"),
 		newPerThreadCounterMetric("flow_wrk", "flows_evicted_total", "", "flows_evicted"),
 		newPerThreadCounterMetric("flow_wrk", "flows_injected_total", "", "flows_injected"),
-		newPerThreadCounterMetric("flow_wrk", "flows_evicted_needs_work", "", "flows_evicted_needs_work"),
+		newPerThreadCounterMetric("flow_wrk", "flows_injected_max_total", "", "flows_injected_max").Optional(),
```

Comment:
```suggestion
		newPerThreadGaugeMetric("flow_wrk", "flows_injected_max", "", "flows_injected_max").Optional(),
```

Other `max` metrics are gauges which is a bit better, because it doesn't make sense to compute a `rate()` on these. Making these counters and using the `_total` suffix can be misleading.